packages/scaffold-ui/test/partials/w3m-account-wallet-features-widget.test.ts

```diff
@@ -0,0 +1,42 @@
+import { W3mAccountWalletFeaturesWidget } from '../../src/partials/w3m-account-wallet-features-widget'
+import { describe, it, expect, vi, beforeAll } from 'vitest'
+import { fixture, elementUpdated } from '@open-wc/testing'
+import { AccountController, CoreHelperUtil } from '@reown/appkit-core'
+import { html } from 'lit'
+import { HelpersUtil } from '../utils/HelpersUtil'
+
+// --- Constants ---------------------------------------------------- //
+const WALLET_FEATURE_WIDGET_TEST_ID = 'w3m-account-wallet-features-widget'
+const MOCK_ADDRESS = '0xcd2a3d9f938e13cd947ec05abc7fe734df8dd826'
+
+describe('W3mLegalCheckbox', () => {
+  beforeAll(() => {
+    vi.spyOn(CoreHelperUtil, 'isMobile').mockReturnValue(false)
+    vi.spyOn(AccountController, 'state', 'get').mockReturnValue({
+      ...AccountController.state,
+      address: undefined
+    })
```

Comment:
There were some issues with valtio proxy states and web components when trying to use `vi.mock('@reown/appkit-core', ()=> ...)`, instead i used `vi.spyOn` which works for now 🙏